src/main/java/core/basesyntax/impl/ProcessDataImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ProcessData;
+import core.basesyntax.transactor.FruitTransaction;
+import core.basesyntax.transactor.Operation;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class ProcessDataImpl implements ProcessData {
+    private static final String COMMA = ",";
+    private static final int HEAD_INFO_INDEX = 1;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int INDEX_OF_AMOUNT = 2;
+    private static final String LINE_SEPARATOR = System.lineSeparator();
```

Comment:
You don't need to create a constant for System.lineSeparator(). It’s already a constant value.